hops/featurestore.py

```diff
@@ -2369,89 +2369,160 @@ def disable_featuregroup_online(featuregroup_name, featuregroup_version=1, featu
     fs_utils._log("Online Feature Serving disabled successfully for featuregroup: {}".format(featuregroup_name))
 
 
-def add_metadata(featuregroup_name, metadata, featuregroup_version=1, featurestore=None):
+def set_featuregroup_tags(name, tags, version=1, featurestore=None):
```

Comment:
why set and not add?

Reply:
I choose the naming "set" as it will overwrite all existing tags. You will send a dict containing {tag1: val2, tag2: val2...}, and it removes all other tags before and sets these new tags you defined. The concept of adding does not include overwriting which set does.

Reply:
Sounds good, I thought it only adds tags